### ISSUE TITLE
MSSQL command hangs with no messages or output

### ISSUE THREAD

**Issue by ysageev** [sequelize/sequelize-auto#392]
I've been trying unsuccessfully for two days to get sequelize-auto to work in any meaningful way with our SQL Server 2016 database.  I have finally gotten to the point where I enter an s-a command and no error messages are output.  However, no files are output, and the app never exits.  It just silently sits there.  

More precisely, It is actually connecting to the db, because I can see "node.js" in the activity monitor.  sequelize-auto appears to be asking the db for its version, but then nothing happens.

My command:
`sequelize-auto -h "xxx.xxx.xxx.xxx" "AAAAA" -e mssql -c "./config.json" -o "./models"`

config.json:
```javascript
{
   "database": "AAAAA",
   "username": "BBBBB",
   "password": "CCCCC",
   "domain": "DDDDD",
   "schema": "EEEEE",
   "dialectModulePath": "sequelize-msnodesqlv8",
   "dialectOptions": {
      "connectionString": "Driver={ODBC Driver 17 for SQL Server};Server=xxx.xxx.xxx.xxx,1433;Network Library=DBMSSOCN;Database=AAAAA;Trusted_Connection=yes;"
   }
}
```

Activity Monitor trace shows that node is running the following query:

```sql
DECLARE @ms_ver NVARCHAR(20); SET @ms_ver = REVERSE(CONVERT(NVARCHAR(20), SERVERPROPERTY('ProductVersion'))); SELECT REVERSE(SUBSTRING(@ms_ver, CHARINDEX('.', @ms_ver)+1, 20)) AS 'version'
```

My expectation is that Sequelizer code for the "EEEEE" schema will be output to the models directory, but there is nothing there.

You will notice that I am using the sequelize-msnodesql package, because if I simply rely on the tedious driver, It simply won't connect:

```javascript
Error [SequelizeConnectionError]: Login failed for user ''.
    at Connection.<anonymous> (C:\Users\USER\AppData\Roaming\npm\node_modules\sequelize-auto\node_modules\sequelize\lib\dialects\mssql\connection-manager.js:109:16)
    at Connection.emit (events.js:223:5)
    at Connection.message (C:\Users\USER\AppData\Roaming\npm\node_modules\tedious\lib\connection.js:2149:18)
    at Connection.dispatchEvent (C:\Users\USER\AppData\Roaming\npm\node_modules\tedious\lib\connection.js:1282:15)
    at MessageIO.<anonymous> (C:\Users\USER\AppData\Roaming\npm\node_modules\tedious\lib\connection.js:1142:14)
    at MessageIO.emit (events.js:223:5)
    at Message.<anonymous> (C:\Users\USER\AppData\Roaming\npm\node_modules\tedious\lib\message-io.js:47:14)
    at Message.emit (events.js:228:7)
    at endReadableNT (C:\Users\USER\AppData\Roaming\npm\node_modules\tedious\node_modules\readable-stream\lib\_stream_readable.js:1094:12)
    at processTicksAndRejections (internal/process/task_queues.js:81:21) {
  name: 'SequelizeConnectionError',
  message: "Login failed for user ''.",
  parent: ConnectionError: Login failed for user ''.
      at ConnectionError (C:\Users\USER\AppData\Roaming\npm\node_modules\tedious\lib\errors.js:13:12)
      at Parser.<anonymous> (C:\Users\USER\AppData\Roaming\npm\node_modules\tedious\lib\connection.js:853:51)
'''

**Comment by billkiddo** [comment]
I managed to publish a success from an Azure managed instance by using a v3: npm install -g sequelize-auto-v3

**Comment by steveschmitt** [comment]
I think this is fixed in `sequelize-auto@0.5.0`  
Please try it and close this issue if so.  Thanks!
